recirq/toric_code/toric_code_rectangle_test.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2022 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Tuple
+
+import cirq
+import numpy as np
+import pytest
+
+from . import toric_code_rectangle as tcr
+
+
+def q(i: int, j: int) -> cirq.GridQubit:
+    return cirq.GridQubit(i, j)
+
+
+class TestToricCodeRectangle:
```

Comment:
can we change this to normal pytest just-a-bunch-of-free-funcitons. for the setup code, you can define a pytest fixture or (imo even better) just factor it out into a helper function and call it at the beginning of each test

Reply:
Done.